README.md

```diff
@@ -123,10 +123,10 @@ sudo make install
 
 **Notes on default of XDG_CONFIG_DIRS**
 
-Dunst uses a different default (${SYSCONFDIR}) for XDG_CONFIG_DIRS at runtime.
+Dunst uses a different default (`${SYSCONFDIR}`) for XDG_CONFIG_DIRS at runtime.
 This is a slight digression from the recommended value in the XDG Base Directory
 Specification (/etc/xdg), because the default config file gets installed to
-${SYSCONFDIR/dunst/dunstrc} to avoid conflicts with /etc/xdg/dunst/dunstrc which
+`${SYSCONFDIR/dunst/dunstrc}` to avoid conflicts with /etc/xdg/dunst/dunstrc which
```

Comment:
Oh, while we're here: The variable is actually named `SYSCONFDIR` and not `SYSCONFDIR/dunst/dunstrc`. (in bash syntax, this would be even weirder :speak_no_evil: )

```suggestion
`${SYSCONFDIR}/dunst/dunstrc` to avoid conflicts with /etc/xdg/dunst/dunstrc which
```